includes/class-cdn-purger.php

```
@@ -0,0 +1,153 @@
+					$ok = false;
+					continue;
+				}
+
+				$code = (int) wp_remote_retrieve_response_code( $response );
+				if ( $code >= 400 ) {
+					$ok = false;
+				}
+			}
+
+			return $ok;
+		}
```

PHPMD (2.15.0)

[error] The method purge_varnish is not named in camelCase. (undefined)

(CamelCaseMethodName)